include/swift/AST/DiagnosticsSema.def

```diff
@@ -2595,6 +2595,8 @@ WARNING(expression_unused_optional_try,none,
         "result of 'try?' is unused", ())
 WARNING(expression_unused_selector_result, none,
         "result of '#selector' is unused", ())
+WARNING(expression_unused_literal,none,
+        "%0 literal is unused", (StringRef))
```

Comment:
We often use `%select{}` for this, so all of the text of the warning is visible in the .def file, but it's not critical.